docs/how-to/integrate-with-machine-agents.md

```diff
@@ -41,6 +41,49 @@ The output should look similar to the contents below:
 Application "jenkins-agent" endpoints [agent] available at "admin/tutorial.jenkins-agent"
 ```
 
+### (Optional) configure external URL for agent discovery and configure websocket
+If you are deploying your jenkins-k8s charm on environments other than microk8s,
```

Comment:
We claim to support multiple k8s distributions, can you refer to kubernetes instead microk8s?